localization/stop_filter/launch/stop_filter.launch.xml

```diff
@@ -3,7 +3,7 @@
   <arg name="input_odom_name" default="ekf_odom"/>
   <arg name="output_odom_name" default="stop_filter_odom"/>
   <arg name="debug_stop_flag" default="debug/stop_flag"/>
-  <node pkg="stop_filter" exec="stop_filter" name="stop_filter" output="screen">
+  <node pkg="stop_filter" exec="stop_filter_node" output="screen">
```

Comment:
Sorry, we have decided to change the log output destination from screen to both, so please make the change. :pray:

Reply:
I have changed the log output from "screen" to "both" in https://github.com/autowarefoundation/autoware.universe/pull/7099/commits/4c41c2dd08351dbc2d269bfa65ec4e208afd84a9

Reply:
I checked again and confirmed that the errors were properly logged to the file.
Thanks!